docs/rules/no-extraneous-dependencies.md

```diff
@@ -19,6 +19,19 @@ You can set the options like this:
 "import/no-extraneous-dependencies": ["error", {"devDependencies": false, "optionalDependencies": false, "peerDependencies": false}]
 ```
 
+You can also use regular expressions instead of literal booleans:
+
+```js
+"import/no-extraneous-dependencies": ["error", {"devDependencies": "test|spec"}]
+```
+
+If you are using JavaScript configuration (e.g., `.eslintrc.js`), then you can use a RegExp literal instead of a string:
+
+```js
+"import/no-extraneous-dependencies": ["error", {"devDependencies": /test|spec/}]
+```
+
+When using a regular expression the result of running [`test`] against the name of the file being linted is used as the boolean value. For example, the above configurations will allow the import of `devDependencies` in files whose names include `test` or `spec`.
```

Comment:
`When using a regular expression` --> `When using a regular expression,` (+ comma)

`the result of running [test] against the name of the file being linted is used as the boolean value.` --> `the setting will be activated if the name of the file being linted matches the given regular expression.`